cmake/shapelib.pc.cmake.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html mentions: " It should typically be a path relative to the installation prefix so that it can be converted to an absolute path in a relocatable way (see CMAKE_INSTALL_FULL_<dir>). However, an absolute path is also allowed."
I'm thinking to the last sentence where the above wouldn't work (and similarly for includedir). I'm aware of people having reporting issues with some distributions that use full paths in those variables.
But we might also have already problems elsewhere in our CMake scripts related to that.
Cf https://github.com/OSGeo/PROJ/pull/3251 for related fixes in the PROJ project

Reply:
I first checked what autotools created and the recreated the path scheme accordingly.

Anyway, might adopt https://github.com/OSGeo/PROJ/pull/3251 then

Reply:
Using absolute paths is a problem already now: The CMake config currently has a hard-coded `include` instead of `CMAKE_INSTALL_INCLUDEDIR`.